RevenueCatUI/PaywallView.swift

```diff
@@ -111,15 +104,8 @@ public struct PaywallView: View {
 
     // @PublicForExternalTesting
     init(configuration: PaywallViewConfiguration, paywallViewOwnsPurchaseHandler: Bool = true) {
-        if paywallViewOwnsPurchaseHandler {
-            self._doNotAccessPurchaseHandler = .init(wrappedValue: configuration.purchaseHandler)
-        } else {
-            // this is unused and is only present to fulfill the need to have an object assigned
-            // to a @StateObject
-            self._doNotAccessPurchaseHandler = .init(wrappedValue: PurchaseHandler.default())
-        }
-
-        self.purchaseHandler = configuration.purchaseHandler
```

Comment:
was it being cleared here?

Reply:
Yeah yeah, this is where it was being cleared unintentionally during a rerender 😬